source/usage-examples/count.txt

```diff
@@ -3,3 +3,47 @@ Count Documents
 ===============
 
 .. default-domain:: mongodb
+
+You can count the number of documents in a collection using the
+``EstimatedDocumentCount()`` and ``CountDocuments()`` methods. 
+
```

Comment:
It might be worth explaining that `EstimatedDocumentCount` returns an approximate count because it uses the traditional `count` command (or `aggregate` with `$collStats` on the newest server versions) and that `count` command can be slightly incorrect when running against sharded clusters or after unclean shutdowns. `CountDocuments`, on the other hand, performs an actual aggregation to count the number of documents. While it can be performance-intensive (especially without a filter), the count will be accurate.

Not sure if that's too much detail for users, but at least qualify `EstimatedDocumentCount` as approximate here and below.

Reply:
As discussed offline, it sounds like the specifics will be in a separate fundamentals page.